CHANGELOG.md

```diff
@@ -2,9 +2,19 @@
 
 ### 2.x.x (In progress)
 
-***SDK***
+**SDK**
+
+- **Fixed:** support of `playsinline` instruction of  `Video` html tag
+
```

Comment:
You don't need an extra space here - this causes separate lists to be rendered so there's an extra gap between the items.